src/sql/workbench/contrib/query/browser/query.contribution.ts

```diff
@@ -287,6 +287,14 @@ KeybindingsRegistry.registerCommandAndKeybindingRule({
 	handler: gridCommands.saveAsJson
 });
 
+KeybindingsRegistry.registerCommandAndKeybindingRule({
+	id: gridActions.GRID_SAVEMARKDOWN_ID,
+	weight: KeybindingWeight.EditorContrib,
+	when: ResultsGridFocusCondition,
+	primary: KeyChord(KeyMod.CtrlCmd | KeyCode.KeyR, KeyMod.CtrlCmd | KeyCode.KeyM),
```

Comment:
Did you verify this isn't being used by anything else?

Reply:
I did a search for `KeyMod.CtrlCmd | KeyCode.KeyR` and there's only one usage with <kbd>ctrl</kbd>+<kbd>m</kbd> following it. Is that sufficient?

Reply:
Yup, looks good.